framework/components/AFeatureDiscovery/components/ATourTooltip.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+
+import {AContainer, ARow, ACol} from "../../ALayout";
+import AButton from "../../AButton";
+import AIcon from "../../AIcon";
+
+const ATourTooltip = ({
+  index,
+  step,
+  isLastStep,
+  onPrev,
+  onClose,
+  onNext,
+  ...tooltipProps
+}) => {
+  return (
+    <div
+      {...tooltipProps}
```

Comment:
From `ATourStep`, I don't see how this could ever be anything other than an empty object